environment.sh

```diff
@@ -35,4 +35,8 @@ export BROADCAST_USER_2_ID='1048af40-45f6-4249-a670-df72ba3352d7'
 export BROADCAST_USER_2_EMAIL='notify-tests-preview+local-broadcast2@digital.cabinet-office.gov.uk'
 export BROADCAST_USER_2_NUMBER=07700900222
 
+export PLATFORM_ADMIN_ID='c3d33860-a967-40cf-8eb4-ec1ee38a4df9'
+export PLATFORM_ADMIN_EMAIL='notify-tests-preview+local-broadcast3@digital.cabinet-office.gov.uk'
+export PLATFORM_ADMIN_NUMBER=07700900333
```

Comment:
Change this to one of the numbers ending "000", "111" or "222".  These three numbers are ignored by Notify, so that the functional tests don't add to our lifetime total limit of 40k SMS messages on the Emergency Alerts Service account